server.js

```diff
@@ -218,11 +218,33 @@ app.get('/live/:year/:week', function(req, res) {
 	});
 })
 
+/**
+ * GET route for the current week in the season. This route can be used when
+ * wanting to know the current season and the current week in the season.
+ */
+ app.get('/currentSeasonAndWeek', function(req, res) {
+	 url = 'http://www.nfl.com/schedules';
+	 request(url, function(error, response, html) {
+		 if (!error) {
+			 console.log(response.statusCode + ' GET ' + url);
+			 var path = response.request.uri.path;
+			 var resultStrings = path.split("/");
+			 var year = resultStrings[2];
+			 var week = resultStrings[3];
+			 week = week.slice(3, week.length);
+			 res.statusCode = 200;
+			 res.send({'year': year, 'week': week});
```

Comment:
This route is only for getting the current season and week in the NFL. So it would return the season as a year and the week as a number between 1 and 17. We need this so the API can automatically create season objects for groups.